docs/explanation/reader-explanation.md

```diff
@@ -0,0 +1,274 @@
+                                   "/dat_mat_components/Temperature 1/value",
+                                   "/dat_mat_components/Temperature 1/unit",
+                                   "/dat_mat_components/LI Demod 1 X/value",
+                                   "/dat_mat_components/LI Demod 1 X/unit",
+                                   "/dat_mat_components/LI Demod 1 Y/value",
+                                   "/dat_mat_components/LI Demod 1 Y/unit",
+                                   "/dat_mat_components/LI Demod 2 X/value",
+                                   "/dat_mat_components/LI Demod 2 X/unit",
+                                   "/dat_mat_components/LI Demod 2 Y/value",
+                                   "/dat_mat_components/LI Demod 2 Y/unit"],
+                              "2": ["/dat_mat_components/Bias [filt]/value",
+                                   "/dat_mat_components/Bias [filt]/unit",
+                                   "dat_mat_components/Bias [filt]/metadata",
+                                   "/dat_mat_components/Current [filt]/value",
+                                   "/dat_mat_components/Current [filt]/unit",
+                                   "/dat_mat_components/Current [filt]/metadata",
+                                   "/dat_mat_components/Temperature 1 [filt]/value",
+                                   "/dat_mat_components/Temperature 1 [filt]/unit",
+                                   "/dat_mat_components/Temperature 1 [filt]/metadata",
+                                   "/dat_mat_components/LI Demod 1 X [filt]/value",
+                                   "/dat_mat_components/LI Demod 1 X [filt]/unit",
+                                   "/dat_mat_components/LI Demod 1 X [filt]/metadata",
+                                   "/dat_mat_components/LI Demod 1 Y [filt]/value",
+                                   "/dat_mat_components/LI Demod 1 Y [filt]/unit",
+                                   "/dat_mat_components/LI Demod 1 Y [filt]/metadata",
+                                   "/dat_mat_components/LI Demod 2 X [filt]/value",
+                                   "/dat_mat_components/LI Demod 2 X [filt]/unit",
+                                   "/dat_mat_components/LI Demod 2 X [filt]/metadata",
+                                   "/dat_mat_components/LI Demod 2 Y [filt]/value",
+                                   "/dat_mat_components/LI Demod 2 Y [filt]/unit",
+                                   "/dat_mat_components/LI Demod 2 Y [filt]/metadata"]}
+}
+```
+**NOTES**
+
+- Each key is nevigating the NeXus concept (e.g. `/ENTRY[entry]/INSTRUMENT[instrument]/piezo_config/active_calib` key nevigates `ENTRY` -> `INSTRUMENT` -> `piezo_config` -> `active_calib` field in `NXsts` application definition.) in NeXus application definition.  
+- Value denoted by `@eln` refers the data must come from eln (user provided), but can be changed if rwa file contains that piece of data. 
```

Comment:
```suggestion
- Each key is pointing to the NeXus concept (e.g. `/ENTRY[entry]/INSTRUMENT[instrument]/piezo_config/active_calib` key nevigates `ENTRY` -> `INSTRUMENT` -> `piezo_config` -> `active_calib` field in `NXsts` application definition.) in the NeXus application definition.  
- If the value is denoted by the token `@eln`, the data must come from the ELN (user provided), but this can be changed if the raw file contains that piece of data as well. 
```

Reply:
Resolved!